broker/src/main/java/io/camunda/zeebe/broker/Broker.java

```diff
@@ -529,7 +539,7 @@ public EmbeddedGatewayService getEmbeddedGatewayService() {
   // only used for tests
   @Deprecated
   public Atomix getAtomix() {
-    return atomix;
+    return testCompanionObject.atomix;
```

Comment:
:question: Do we need the Atomix object in tests? Ideally we could reuse the cluster services. Well, ideally we would have a good monitoring API so we wouldn't need to check the internals, but baby steps :smile:

Reply:
Yes we do, in two places: `CommandApiRule` (has dependencies to Atomix, but not to Broker classes) and `ClusertingRule` (wants to access the `UnicastService`)

I think one can take a closer look at both places, but for now these are the ones that still need it.